lib/chef/provider/package/yum.rb

```diff
@@ -29,8 +29,8 @@ class Package
       class Yum < Chef::Provider::Package
         include Chef::Mixin::GetSourceFromPackage
 
-        provides :package, platform_family: %w{rhel fedora amazon}
-        provides :yum_package, os: "linux"
+        provides :package, platform_family: %w{rhel fedora amazon aix}
```

Comment:
package needs to be bff_package on AIX so this line needs to be reverted, you must use `yum_package` explicitly on AIX